src/catalog.h

```diff
@@ -581,20 +582,62 @@ enum Anum_bgw_job_stat_pkey_idx
 #define Natts_bjw_job_stat_pkey_idx \
 	(_Anum_bgw_job_stat_pkey_idx_max - 1)
 
+/******************************
+ *
+ * installation_metadata table definitions
+ *
+ ******************************/
+
+#define INSTALLATION_METADATA_TABLE_NAME		"installation_metadata"
+
+enum Anum_installation_metadata
+{
+	Anum_installation_metadata_key = 1,
+	Anum_installation_metadata_value,
+	_Anum_installation_metadata_max,
+};
+
+#define Natts_installation_metadata \
+	(_Anum_installation_metadata_max - 1)
+
+typedef struct FormData_installation_metadata
+{
+	NameData	key;
+	text	   *value;
+}			FormData_installation_metadata;
```

Comment:
Why does `pg_indent` do this....? Whenever it does this, I usually remove the change, but not sure what convention is for master.

Reply:
I've found this happens when `pg_indent` doesn't know about a type name

Reply:
Yeah, it happens when there is no actual object reference of this type in the code.